reps/config.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  rootPath: "../../mozilla-central",
+  devtoolsPath: "../../mozilla-central/devtools",
```

Comment:
My meaning is we can do
```
'use strict';

module.exports = {
  rootPath: "../../../../mozilla.org/",
  devtoolsPath: "../../../../mozilla.org/fx-team/devtools",
};
```
Then developers can config their paths in `config.js` for running Rep Tester. It might be easier than changing directory structure.